docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
```

Comment:
I think this docker-compose file it's not needed, otherwise it deserves another position in the repository (e.g. inside `resources` folder)

Reply:
Removed. This is not needed. Thank you.